### DISCUSSION TITLE
Need "Skip All" button

### DISCUSSION THREAD

**Discussion by Naghual** [doublecmd/doublecmd#1429] [Suggestions]
Copying a lot of files with symbolic links gets me a lot of pain because of absence of "Skip all" button.
![image](https://github.com/doublecmd/doublecmd/assets/1244173/e23c4930-52cd-4593-bc86-fde37b3c727c)

**Comment by alexx2000** [comment]
As a workaround you can enable [Skip file operations errors and write them to log window](https://doublecmd.github.io/doc/en/configuration.html#ConfigOperations) option.

**Comment by teqexplorer** [comment]
This is a crazy/enormous time-waster, especially for people using a lot of symlinks to help our efficiency in navigating ever-increasing file repositories. Please provide a Skip All for symlinks.

**Comment by alexx2000** [comment]
DC already has `Skip All` button.
